com.unity.multiplayer.mlapi/Editor/PostProcessScene.cs

```diff
@@ -11,7 +11,9 @@ public class NetworkScenePostProcess : MonoBehaviour
         [PostProcessScene(int.MaxValue)]
         public static void ProcessScene()
         {
-            var traverseSortedObjects = FindObjectsOfType<NetworkObject>().ToList();
+            // find all scene objects that have not been spawned yet
+            // TODO: long term, replace the means of finding candidate objects to repace FindObjectsOfType
+            var traverseSortedObjects = FindObjectsOfType<NetworkObject>().Where(x => x.IsSceneObject == null).ToList();
```

Comment:
coming from: #629 — fix: Fix object traversal assigning instanceIds to none scene objects